6/README.md

```diff
@@ -0,0 +1,1078 @@
+
+This Specification is a [free and open standard](http://www.digistan.org/open-standard:definition) and is governed by the Digital Standards Organization's [Consensus-Oriented Specification System](http://www.digistan.org/spec:1/COSS).
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in [RFC 2119](http://tools.ietf.org/html/rfc2119).
+
+## Related RFC specifications (TBD)
+- X/Device Model 
+- X/Attributes 
+- X/Properties 
+
+## Tango Database specification
+
+This specification is intended to formally document the Tango Database. 
+Runtime implementations of the Database in conforming Tango implementations MUST follow this specification.
+
+
+### Goals of the Database
+* provide a means of defining and querying devices.
+* provide means for device servers to be configured from central control
+* serve as nameserver such that the Devices are accessible by name
+
+### Features of the Database
+
+* store configuration data (properties) used at startup of a device server 
+* act as nameserver by storing the dynamic network addresses
+* act as permanent store of dynamic settings which need to be memorized
+* ensure the uniqueness of device name and of aliases
+* keep list of controlled servers which are to be started and run on a particular host
+* limited set functionality provide by file as alternative to network database
+
+### Use Cases of the Database
+
+* register a new device server
+* stores the last user setting point of the current provided by a power supply
+* define persistent configuration properties to be used by a camera every time it starts up
+* allow a subscriber or client to get connect to a stepper motor
+* define properties for all instances of a class of vaccuum pump
+* check if a electrometer is online or not
+* a user would like to rollback the configuration of a Device when its initialization is failing
+* remove "empty" Devices from Database to clean it up
+* get all Devices of a given class of temperature gauges
+* user wants to see the history (last ten values) of a Property
+* a monitor tool wants the Info of a not running detector Device (exported, host, server)
+* a boot-up process wants to get Starter Level configuration for a list of Devices
+
+## Specification
+The database SHALL provide a means of defining and querying devices.
+
```

Comment:
* As the Tango database service is a central service of the Tango control system, it SHALL be designed to be highly available and support a high level of client requests (e.g we have peaks of 3500 SQL requests/ s at SOLEIL on the accelerators control system)